plugins/meta/firewall/README.md

```diff
@@ -133,3 +133,17 @@ of the container as shown:
 - `-s 10.88.0.2 -m conntrack --ctstate RELATED,ESTABLISHED -j CNI-FORWARD`
 - `-d 10.88.0.2 -j CNI-FORWARD`
 
+A new chain, CNI-ADMIN is added to the CNI-FORWARD chain. CNI-ADMIN is intended as a user-controlled chain that all traffic passes through prior to the rules managed by the `firewall` plugin.
```

Comment:
@xtreme-sameer-vohra thanks for the PR! Just a couple comments...

The CNI-ADMIN chain and the CNI-FORWARD chain are actually orthogonal. What I think you really mean is that CNI-ADMIN is added to the FILTER table with precedence over CNI-FORWARD. So perhaps:

`A new chain CNI-ADMIn is added to the FILTER table with precedence over any other CNI firewall plugin chains. CNI-ADMIN is intended ... (the rest of what you have here)`

Reply:
I might have misunderstood, but based on [this section](https://github.com/containernetworking/plugins/blob/master/plugins/meta/firewall/iptables.go#L168-L169) it looks like `CNI-ADMIN` is added to the `CNI-FORWARD` chain

Reply:
Although, it would effectively have the same impact as having precedence over any other rule in the CNI-FORWARD chain

Reply:
@xtreme-sameer-vohra chains aren't added to other chains. iptables chains are completely independent, but you can jump from chain to chain. But that's not what's happening here. It's simply that packets *first* get sent to the CNI-ADMIN chain (becuase it's ordered first in the table) and if no rule in that chain matches, then they get sent to the CNI-FORWARD chain.

So all I'd like changed here is the wording so that it accurately represents what happens in iptables' logic. Thoughts? Thanks!

Reply:
Thanks for the clarification. I updated it to reflect the entry for `CNI-FORWARD`, referencing the `FORWARD` chain rather than the `FILTER` table. Does that make sense ?